test/ModelTest.hs

```
@@ -173,9 +173,11 @@ test_7 = Test.stepsWithTransaction "test_7" $ \step cn2 -> do
     -- TODO: should be lookup auth on rootParty
     vol <- runReaderT (addVolumeSetPrivate aiAcct) aiCtxt
     step "Then the public can't view it"
-    -- Implementation of getVolume PUBLIC
-    mVolForAnon <- runWithNoIdent cn2 (lookupVolume ((volumeId . volumeRow) vol))
-    mVolForAnon @?= Nothing
+    mVol <-
+        runWithNoIdent
+            cn2
+            (getVolume PermissionREAD ((volumeId . volumeRow) vol))
+    mVol @?= LookupFailed
```

I would leave these looser, as not lookupFound

---

I like making the tests prickly... "fail fast"